src/opera/instance/node.py

```diff
@@ -131,7 +131,16 @@ def get_attribute(self, params):
         if attr in self.attributes:
             return self.attributes[attr].eval(self)
 
-        # TODO(@tadeboro): Add capability access.
+        # If we have no attribute, try searching for capability.
+        capabilities = tuple(
+            c for c in self.template.capabilities if c.name == attr
+        )
+        if len(capabilities) > 1:
+            self.error = DataError("More than one capability is named '{}'.".format(attr, ))
+            raise self.error
```

Comment:
Lines length exceeds 78 chars

Reply:
Aha you were probably referring to [PEP8](https://www.python.org/dev/peps/pep-0008/) where lines should be limited to a maximum of 79 characters.

Reply:
This was also refactored and that line was limited to 79 chars.